features/claims/advocate/scheme_sixteen/advocate_supplementary_claim_submit.feature

```diff
@@ -0,0 +1,154 @@
+    When I click "Continue" in the claim form
+    Then I should be in the 'Miscellaneous fees' form page
+    And I should see the advocate categories 'Junior,Leading junior,KC'
+    And the following miscellaneous fee checkboxes should exist:
+      | section       | fee_description                          |
+      | miscellaneous | Additional preparation fee               |
+      | miscellaneous | Confiscation hearings (half day)         |
+      | miscellaneous | Confiscation hearings (half day uplift)  |
+      | miscellaneous | Confiscation hearings (whole day)        |
+      | miscellaneous | Confiscation hearings (whole day uplift) |
+      | miscellaneous | Deferred sentence hearings               |
+      | miscellaneous | Deferred sentence hearings uplift        |
+      | miscellaneous | Paper heavy case                         |
+      | miscellaneous | Plea and trial preparation hearing       |
+      | miscellaneous | Section 28 hearing                       |
+      | miscellaneous | Sentence hearings                        |
+      | miscellaneous | Sentence hearings uplift                 |
+      | miscellaneous | Special preparation fee                  |
+      | miscellaneous | Standard appearance fee                  |
+      | miscellaneous | Standard appearance fee uplift           |
+      | miscellaneous | Unused materials (over 3 hours)          |
+      | miscellaneous | Unused materials (up to 3 hours)         |
+      | miscellaneous | Wasted preparation fee                   |
+
+    When I select an advocate category of 'Junior'
+    And I choose the 'Additional preparation fee' miscellaneous fee with quantity of '1'
+    And I choose the 'Confiscation hearings (half day)' miscellaneous fee with quantity of '2'
+    And I choose the 'Confiscation hearings (half day uplift)' miscellaneous fee with quantity of '1'
+    And I choose the 'Section 28 hearing' miscellaneous fee with quantity of '1'
+    And I choose the 'Standard appearance fee' miscellaneous fee with quantity of '2'
+    And I choose the 'Standard appearance fee uplift' miscellaneous fee with quantity of '1'
+    And I choose the 'Wasted preparation fee' miscellaneous fee with quantity of '1'
+
+    And I save and open screenshot
+    Then the following supplementary fee details should exist:
+      | section       | fee_description                         | rate   | hint                            | help |
+      | miscellaneous | Additional preparation fee              | 62.00  | Number of fees                  | true |
+      | miscellaneous | Confiscation hearings (half day)        | 151.00 | Number of half days             | true |
+      | miscellaneous | Confiscation hearings (half day uplift) | 60.40  | Number of additional defendants | true |
+      | miscellaneous | Section 28 hearing                      | 670.00 | Number of hearings              | true |
```

Comment:
should this be 1000.00?

Reply:
For now, I'm not getting the new value of the S28 fee through into my local environment. This is set for 670 for the moment to stop it shouting, but I've got a not (on the jira ticket) that this will need changing to 1000.00 once we're able to link things up and do a proper test.

Reply:
We could deploy Fee Calculator, which would allow this to get the correct value and you can update the VCR file.